rollup.config.js

```diff
@@ -0,0 +1,95 @@
+import babel from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs';
+import nodeResolve from '@rollup/plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+import copy from 'rollup-plugin-copy';
+import esbuild from 'rollup-plugin-esbuild';
+import scss from 'rollup-plugin-scss';
+import * as sass from 'sass';
+
+const serverBundle = {
+	input: 'src/server/app.js',
+	output: {
+		file: 'built/main.js',
+		sourcemap: 'inline'
+	},
+	external: [
+		'express',
+		'express-session',
+		'node:http',
+		'prop-types',
+		'react',
+		'react-bootstrap-typeahead',
+		'react-dom/server',
+		'react-helmet',
+		'react-redux',
+		'react-router-dom',
+		'react-router-dom/server.js',
+		'redux',
+		'redux-thunk'
+	],
```

Comment:
This is to omit the named modules from the bundle and only include them at runtime, as detailed here: https://rollupjs.org/troubleshooting/#warning-treating-module-as-external-dependency.

It also prevents `(!) Unresolved dependencies` warnings in the build process:

```
src/server/app.js → built/main.js...
(!) Unresolved dependencies
https://rollupjs.org/troubleshooting/#warning-treating-module-as-external-dependency
node:http (imported by "src/server/app.js")
node:path (imported by "src/server/app.js")
node:url (imported by "src/server/app.js")
express (imported by "src/server/app.js", "src/server/api-router.js" and "src/server/router.js")
express-session (imported by "src/server/app.js")
morgan (imported by "src/server/app.js")
serve-favicon (imported by "src/server/app.js")
react-helmet (imported by "src/server/router.js", "src/react/Layout.jsx", "src/react/components/InstanceDocumentTitle.jsx" and "src/react/components/ErrorMessage.jsx")
react-router-dom (imported by "src/server/router.js", "src/react/AppRoutes.jsx", "src/react/Layout.jsx", "src/react/components/Navigation.jsx", "src/react/components/InstanceLink.jsx", "src/react/components/Header.jsx" and "src/react/components/SearchBar.jsx")
redux (imported by "src/server/router.js")
redux-thunk (imported by "src/server/router.js")
react (imported by "src/react/react-html.jsx", "src/react/AppRoutes.jsx", "src/react/pages/Home.jsx", "src/react/pages/NotFound.jsx", "src/react/Layout.jsx", "src/react/pages/instances/Award.jsx", "src/react/pages/instances/FestivalSeries.jsx", "src/react/pages/instances/Festival.jsx", "src/react/pages/lists/Seasons.jsx", "src/react/pages/lists/Companies.jsx", "src/react/pages/lists/AwardCeremonies.jsx", "src/react/pages/lists/Festivals.jsx", "src/react/pages/instances/AwardCeremony.jsx", "src/react/pages/lists/Venues.jsx", "src/react/pages/instances/Character.jsx", "src/react/pages/instances/Company.jsx", "src/react/pages/lists/FestivalSerieses.jsx", "src/react/pages/lists/Awards.jsx", "src/react/pages/lists/Productions.jsx", "src/react/pages/instances/Season.jsx", "src/react/pages/lists/Materials.jsx", "src/react/page-wrappers/ListPageWrapper.jsx", "src/react/pages/instances/Venue.jsx", "src/react/pages/lists/Characters.jsx", "src/react/pages/lists/People.jsx", "src/react/pages/instances/Material.jsx", "src/react/page-wrappers/InstancePageWrapper.jsx", "src/react/pages/instances/Production.jsx", "src/react/pages/instances/Person.jsx", "src/react/components/AppendedEmployerCompany.jsx", "src/react/components/AppendedCoEntities.jsx", "src/react/components/CharactersList.jsx", "src/react/components/CreativeProductionsList.jsx", "src/react/components/CommaSeparatedMaterials.jsx", "src/react/components/AppendedEntities.jsx", "src/react/components/AppendedRoles.jsx", "src/react/components/AppendedMembers.jsx", "src/react/components/AppendedVenue.jsx", "src/react/components/CommaSeparatedProductions.jsx", "src/react/components/AppendedQualifier.jsx", "src/react/components/AppendedFormatAndYear.jsx", "src/react/components/CrewProductionsList.jsx", "src/react/components/AppendedDepictions.jsx", "src/react/components/AppendedPerformers.jsx", "src/react/components/AppendedDate.jsx", "src/react/components/CommaSeparatedInstanceLinks.jsx", "src/react/components/Entities.jsx", "src/react/components/AppendedProductionTeamCredits.jsx", "src/react/components/AppendedProductionDates.jsx", "src/react/components/InstanceDocumentTitle.jsx", "src/react/components/FestivalLinkWithContext.jsx", "src/react/components/Footer.jsx", "src/react/components/JoinedRoles.jsx", "src/react/components/ListWrapper.jsx", "src/react/components/PageSubtitle.jsx", "src/react/components/ProducerProductionsList.jsx", "src/react/components/ErrorMessage.jsx", "src/react/components/Navigation.jsx", "src/react/components/ProducerCredits.jsx", "src/react/components/ProducerEntities.jsx", "src/react/components/InstanceFacet.jsx", "src/react/components/PageTitle.jsx", "src/react/components/MaterialLinkWithContext.jsx", "src/react/components/InstanceLink.jsx", "src/react/components/Header.jsx", "src/react/components/InstanceLabel.jsx", "src/react/components/InstanceLinksList.jsx", "src/react/components/PrependedSurInstance.jsx", "src/react/components/MaterialsList.jsx", "src/react/components/ProductionLinkWithContext.jsx", "src/react/components/WritingCredits.jsx", "src/react/components/ProductionsList.jsx", "src/react/components/ProductionTeamCreditsList.jsx", "src/react/components/VenueLinkWithContext.jsx", "src/react/components/WritingEntities.jsx" and "src/react/components/SearchBar.jsx")
react-dom/server (imported by "src/react/react-html.jsx")
react-redux (imported by "src/react/react-html.jsx", "src/react/Layout.jsx", "src/react/pages/instances/Award.jsx", "src/react/pages/instances/FestivalSeries.jsx", "src/react/pages/instances/Festival.jsx", "src/react/pages/lists/Seasons.jsx", "src/react/pages/lists/Companies.jsx", "src/react/pages/lists/AwardCeremonies.jsx", "src/react/pages/lists/Festivals.jsx", "src/react/pages/instances/AwardCeremony.jsx", "src/react/pages/lists/Venues.jsx", "src/react/pages/instances/Character.jsx", "src/react/pages/instances/Company.jsx", "src/react/pages/lists/FestivalSerieses.jsx", "src/react/pages/lists/Awards.jsx", "src/react/pages/lists/Productions.jsx", "src/react/pages/instances/Season.jsx", "src/react/pages/lists/Materials.jsx", "src/react/pages/instances/Venue.jsx", "src/react/pages/lists/Characters.jsx", "src/react/pages/lists/People.jsx", "src/react/pages/instances/Material.jsx", "src/react/pages/instances/Production.jsx" and "src/react/pages/instances/Person.jsx")
react-router-dom/server.js (imported by "src/react/react-html.jsx")
prop-types (imported by "src/react/Layout.jsx", "src/react/pages/instances/Award.jsx", "src/react/pages/instances/FestivalSeries.jsx", "src/react/pages/instances/Festival.jsx", "src/react/pages/lists/Seasons.jsx", "src/react/pages/lists/Companies.jsx", "src/react/pages/lists/AwardCeremonies.jsx", "src/react/pages/lists/Festivals.jsx", "src/react/pages/instances/AwardCeremony.jsx", "src/react/pages/lists/Venues.jsx", "src/react/pages/instances/Character.jsx", "src/react/pages/instances/Company.jsx", "src/react/pages/lists/FestivalSerieses.jsx", "src/react/pages/lists/Awards.jsx", "src/react/pages/lists/Productions.jsx", "src/react/pages/instances/Season.jsx", "src/react/pages/lists/Materials.jsx", "src/react/page-wrappers/ListPageWrapper.jsx", "src/react/pages/instances/Venue.jsx", "src/react/pages/lists/Characters.jsx", "src/react/pages/lists/People.jsx", "src/react/pages/instances/Material.jsx", "src/react/page-wrappers/InstancePageWrapper.jsx", "src/react/pages/instances/Production.jsx", "src/react/pages/instances/Person.jsx", "src/react/components/AppendedEmployerCompany.jsx", "src/react/components/AppendedCoEntities.jsx", "src/react/components/CharactersList.jsx", "src/react/components/CreativeProductionsList.jsx", "src/react/components/CommaSeparatedMaterials.jsx", "src/react/components/AppendedEntities.jsx", "src/react/components/AppendedRoles.jsx", "src/react/components/AppendedMembers.jsx", "src/react/components/AppendedVenue.jsx", "src/react/components/CommaSeparatedProductions.jsx", "src/react/components/AppendedQualifier.jsx", "src/react/components/AppendedFormatAndYear.jsx", "src/react/components/CrewProductionsList.jsx", "src/react/components/AppendedDepictions.jsx", "src/react/components/AppendedPerformers.jsx", "src/react/components/AppendedDate.jsx", "src/react/components/CommaSeparatedInstanceLinks.jsx", "src/react/components/Entities.jsx", "src/react/components/AppendedProductionTeamCredits.jsx", "src/react/components/AppendedProductionDates.jsx", "src/react/components/InstanceDocumentTitle.jsx", "src/react/components/FestivalLinkWithContext.jsx", "src/react/components/JoinedRoles.jsx", "src/react/components/ListWrapper.jsx", "src/react/components/PageSubtitle.jsx", "src/react/components/ProducerProductionsList.jsx", "src/react/components/ErrorMessage.jsx", "src/react/components/ProducerCredits.jsx", "src/react/components/ProducerEntities.jsx", "src/react/components/InstanceFacet.jsx", "src/react/components/PageTitle.jsx", "src/react/components/MaterialLinkWithContext.jsx", "src/react/components/InstanceLink.jsx", "src/react/components/InstanceLabel.jsx", "src/react/components/InstanceLinksList.jsx", "src/react/components/PrependedSurInstance.jsx", "src/react/components/MaterialsList.jsx", "src/react/components/ProductionLinkWithContext.jsx", "src/react/components/WritingCredits.jsx", "src/react/components/ProductionsList.jsx", "src/react/components/ProductionTeamCreditsList.jsx", "src/react/components/VenueLinkWithContext.jsx" and "src/react/components/WritingEntities.jsx")
react-bootstrap-typeahead (imported by "src/react/components/SearchBar.jsx")
```